From 665f29de88bdeeb504b1ccdc27517403149f7238 Mon Sep 17 00:00:00 2001
From: William Roy <wroy@proton.me>
Date: Thu, 19 Sep 2024 15:41:45 -0400
Subject: [PATCH] feat(hues): reduce chroma level

---
 lua/mini/hues.lua | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lua/mini/hues.lua b/lua/mini/hues.lua
index 8bfb033..2792ab9 100644
--- a/lua/mini/hues.lua
+++ b/lua/mini/hues.lua
@@ -329,7 +329,7 @@ MiniHues.make_palette = function(config)
   local l_mid = 0.5 * (bg_l + fg_l)
 
   -- Configurable chroma level
-  local chroma = ({ low = 4, medium = 8, high = 16 })[saturation]
+  local chroma = ({ low = 4, medium = 5, high = 15 })[saturation]
 
   -- Hues
   local hues = H.make_hues(bg_lch.h, fg_lch.h, n_hues)
-- 
2.46.0

